docs/quick-start.md

```diff
@@ -97,17 +97,17 @@ helm install --wait -n kubewarden kubewarden-defaults kubewarden/kubewarden-defa
 ```
 
 :::caution
-Since [`v0.4.0`](https://github.com/kubewarden/kubewarden-controller/releases/tag/v0.4.0), a `PolicyServer` resource named `default` will not be created using the `kubewarden-controller` chart.
+Since [`v0.4.0`](https://github.com/kubewarden/kubewarden-controller/releases/tag/v0.4.0), a `[[< policy-server-component >]]` resource named `default` will not be created using the `kubewarden-controller` chart.
```

Comment:
@kubewarden/kubewarden-developers  can we drop this caution block already?

Reply:
Just using the quick start for examples of how this might work for now.